pkg/scheduler/scheduler_test.go

```diff
@@ -3815,61 +3815,21 @@ func TestScheduleForTAS(t *testing.T) {
 			Ready().
 			Obj(),
 	}
-	defaultSingleLevelTopology := kueuealpha.Topology{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "tas-single-level",
-		},
-		Spec: kueuealpha.TopologySpec{
-			Levels: []kueuealpha.TopologyLevel{
-				{
-					NodeLabel: corev1.LabelHostname,
-				},
-			},
-		},
-	}
-	defaultTwoLevelTopology := kueuealpha.Topology{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "tas-two-level",
-		},
-		Spec: kueuealpha.TopologySpec{
-			Levels: []kueuealpha.TopologyLevel{
-				{
-					NodeLabel: tasRackLabel,
-				},
-				{
-					NodeLabel: corev1.LabelHostname,
-				},
-			},
-		},
-	}
-	defaultFlavor := kueue.ResourceFlavor{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "default",
-		},
-		Spec: kueue.ResourceFlavorSpec{},
-	}
-	defaultTASFlavor := kueue.ResourceFlavor{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "tas-default",
-		},
-		Spec: kueue.ResourceFlavorSpec{
-			NodeLabels: map[string]string{
-				"tas-node": "true",
-			},
-			TopologyName: ptr.To[kueue.TopologyReference]("tas-single-level"),
-		},
-	}
-	defaultTASTwoLevelFlavor := kueue.ResourceFlavor{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "tas-default",
-		},
-		Spec: kueue.ResourceFlavorSpec{
-			NodeLabels: map[string]string{
-				"tas-node": "true",
-			},
-			TopologyName: ptr.To[kueue.TopologyReference]("tas-two-level"),
-		},
-	}
+	defaultSingleLevelTopology := *utiltesting.MakeTopology("tas-single-level").
+		Levels([]string{corev1.LabelHostname}).
+		Obj()
+	defaultTwoLevelTopology := *utiltesting.MakeTopology("tas-two-level").
+		Levels([]string{tasRackLabel, corev1.LabelHostname}).
+		Obj()
+	defaultFlavor := *utiltesting.MakeResourceFlavor("default").Obj()
+	defaultTASFlavor := *utiltesting.MakeResourceFlavor("tas-default").
+		NodeLabel("tas-node", "true").
+		TopologyName("tas-single-level").
+		Obj()
+	defaultTASTwoLevelFlavor := *utiltesting.MakeResourceFlavor("tas-default").
+		NodeLabel("tas-node", "true").
+		TopologyName("tas-two-level").
+		Obj()
```

Comment:
Can we commonize this and use across the codebase as mentioned in the comment?
https://github.com/kubernetes-sigs/kueue/issues/3662#issuecomment-2514525520

Reply:
I suggest to leave this for a follow up, as this is not the aim of the PR